docker-compose.yml

```diff
@@ -86,6 +86,8 @@ services:
     volumes:
       - ${LIBRETIME_CONFIG_FILEPATH:-./config.yml}:/etc/libretime/config.yml:ro
       - libretime_storage:/srv/libretime
+    environment:
+      LIBRETIME_GENERAL_PUBLIC_URL: http://nginx:8080
```

Comment:
Hmm this feels risky, could we pass the env vars only when we run the bulk import ?

Reply:
The suggestion below makes sense rather than changing the compose. The documentation helps ensure that the right URL is set.